### ISSUE TITLE
Language picker improvements

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4580]
The "lang-select-button" is actually not a button but a link. 
It an anchor-element (<a>) with an href, but it does not open a new page when clicking it. It opens a part of the page, like an expandable, collapsible, details/summary or something alike. This is not navigation but an action. I would advise using a button to perform actions so users know what to expect.
More information on the usage of links vs buttons can be found here: https://marcysutton.com/links-vs-buttons-in-modern-web-applications

To follow the WCAG standard, the button/link should also indicate that it opens something. This can be done with `aria-expanded`. When the section is closed, the link should be set to `aria-expanded="false"`, and when it's open, it should be set to `aria-expanded="true"`. This way the state of the section is communicated in the code.
More information on implementation can be found here: https://www.w3.org/TR/wai-aria-1.1/#aria-expanded

What is also confusing, is that you have a language picker that is named after the language. It does not indicate that it is indeed a language picker. Renaming it might clarify usage.

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos

**Comment by rafalguzniczak** [comment]
Hi, I would like to work on this issue! :) Can someone assign me please?

**Comment by rafalguzniczak** [comment]
I prepared [PR](https://github.com/electron/electronjs.org/pull/4638) - I'd be grateful if someone would look at it :) 
I tested it with VoiceOver and it looks fine for me 👍🏻
Also, it would be great if someone could add `#hacktoberfest` label to this issue - thanks!
